### ISSUE TITLE
please support windows！

### ISSUE THREAD

**Issue by ssyangchn** [plexdrive/plexdrive#374]
please support windows

**Comment by felixbuenemann** [comment]
This has been discussed before, it is up to the fuse package used to implement Windows support.

Currently bazil.org/fuse doesn't support Windows, if you know a Go FUSE package that does, let me know.

**Comment by hitech95** [comment]
On windows there is Google Drive Stream Fs

**Comment by felixbuenemann** [comment]
> On windows there is Google Drive Stream Fs

What is your point?

**Comment by hitech95** [comment]
On windows there is already a FS driver directly from google, it work quite similar to fuse.
It create a drive letter and you can stream download files. It have disk cache, etc.

So if he need plexdrive for windows it could try to use that.

**Comment by felixbuenemann** [comment]
Yes, it is available for macOS as well. I use it to store some of my work related files, but haven't tried out how well it works for streaming video.

**Comment by hitech95** [comment]
YEa it works fine, at least better that gdrive-ocamelfuse.
I'm using that as a network share for my Plex docker.
It have drive cache, and I'll mvoe to pledrive once the mmap branch has been merged :D
